### PR TITLE
Add webview for when dvc is not available or not initialized

### PR DESCRIPTION
**Pull request by sroy3**
A start on #2776 

https://user-images.githubusercontent.com/3683420/204931601-e09f795c-3095-4479-b6f9-f8e6b49650f1.mov

### CODE REVIEW

**Comment by sroy3** [comment]
The structure should change a lot in the following PRs. Not sure if it's worth adding tests for now. I'll add them if you think otherwise.

I'll also follow-up for the name and icons of the webview and change the order of the views in the sidebar.

File: extension/src/webview/workspace.ts [hunk 1]
```diff
@@ -48,5 +66,15 @@ export abstract class BaseWorkspaceWebviews<
     return overrideRoot || (await this.getFocusedOrOnlyOrPickProject())
   }
 
+  protected async showEmptyWebview(viewColumn?: ViewColumn) {
+    await createWebview(
```
Comment (sroy3) [inline extension/src/webview/workspace.ts:70]: Creating a simple webview for now, but I've already started creating a class for it to add messages.

File: extension/package.json [hunk 2]
```diff
@@ -1351,7 +1351,7 @@
       {
         "view": "dvc.views.webviews",
         "contents": "[Show Experiments](command:dvc.showExperiments)\n[Show Plots](command:dvc.showPlots)\n[Show Experiments and Plots](command:dvc.showExperimentsAndPlots)",
-        "when": "dvc.commands.available && dvc.project.available"
+        "when": "true"
```
Comment (mattseddon) [inline extension/package.json:1354]: [Q] Would it be easier to show the same view but with different underlying commands under different circumstances? Would that simplify the code elsewhere?

i.e 
```
      {
        "view": "dvc.views.webviews",
        "contents": "[Show Experiments](command:dvc.showExperiments)\n[Show Plots](command:dvc.showPlots)\n[Show Experiments and Plots](command:dvc.showExperimentsAndPlots)",
        "when": "dvc.commands.available && dvc.project.available"
},
      {
        "view": "dvc.views.webviews",
        "contents": "[Show Experiments](command:dvc.showEmptyWebview)\n[Show Plots](command:dvc.showEmptyWebview)\n[Show Experiments and Plots](command:dvc.showEmptyWebview)",
        "when": "!dvc.commands.available || !dvc.project.available"
}
```

**Review by mattseddon — APPROVED** [review]
Left one [Q] that might help to simplify code paths. Just need to check that the new webview is correctly bundled with the extension.

File: extension/package.json (hunk 2 above)
Comment (sroy3) [reply extension/package.json:1354]: I think you're right. I'll try that. Thanks!

**Pull request merged**